sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ExpressionParserSuite.scala

```diff
@@ -435,8 +435,7 @@ class ExpressionParserSuite extends AnalysisTest {
     val intervalLiteral = Literal(CalendarInterval.fromString("interval 3 month 1 hour"))
     assertEqual("InterVal 'interval 3 month 1 hour'", intervalLiteral)
     assertEqual("INTERVAL '3 month 1 hour'", intervalLiteral)
-    assertEqual("Interval 'interval 3 monthsss 1 hoursss'",
-      Literal(null, CalendarIntervalType))
+    intercept("Interval 'interval 3 monthsss 1 hoursss'")
```

Comment:
Can you check the error message?

Reply:
This test suite defines the `intercept` method and uses it in many places. I don't want to break the rule here. It's definitely better to update the `intercept` method and take a error message parameter. This should be done in a followup.

Reply:
the method has `messages` arg

Reply:
damn, the date and timestamp tests don't specify message, and I thought it doesn't have the message arg... Let me update it now.